src/django_components/util/template_tag.py

```diff
@@ -11,152 +16,36 @@
 from django_components.util.tag_parser import TagAttr, parse_tag
 
 
-@dataclass
-class TagSpec:
```

Comment:
`TagSpec` was removed. the `tag`, `end_tag`, `flags` and signature are now all defined by the `BaseNode` subclasses.